frontend/src/metabase/admin/tasks/containers/Help.jsx

```diff
@@ -127,6 +127,15 @@ export default class Help extends Component {
           <AdminHeader title={t`Diagnostic Info`} className="mb2" />
           <p>{t`Please include these details in support requests. Thank you!`}</p>
           <InfoBlock>{detailString}</InfoBlock>
+          <p>{t`Advanced Details (click to download)`}</p>
+          <ul>
+            <li>
+              <a
```

Comment:
Suggestion: use `ExternalLink` component.

Reply:
OK, I switched to using the HelpLink like elsewhere on that page.  Works OK, but the spacing isn't quite right.

<img width="456" alt="Screen Shot 2021-04-21 at 9 42 31 AM" src="https://user-images.githubusercontent.com/3521562/115572922-f2a23980-a285-11eb-95f6-5f2f3ba65e9d.png">

Reply:
Could be the `ol`. Someone more knowledgeable with CSS  like @kdoh or @daltojohnso probably can help with that.

Reply:
Added @daltojohnso  as a reviewer to opine.  Thanks.